src/Classes/ItemsTab.lua

```diff
@@ -2011,6 +2014,22 @@ function ItemsTabClass:EditDisplayItemText(alsoAddItem)
 	main:OpenPopup(500, 500, self.displayItem and "Edit Item Text" or "Create Custom Item from Text", controls, nil, "edit")
 end
 
+-- Opens trade site link for selected item
+function ItemsTabClass:OpenTradeLink()
+	local item = self.displayItem.rawLines
+	local tradeQuery = new("TradeQueryCurItem",item)
+	local url = tradeQuery:ParseItem()
+	os.execute("rundll32 url.dll,FileProtocolHandler " .. url)
```

Comment:
Don't use `os.execute`.  Use `OpenUrl` instead.

Reply:
swapped to OpenURL